### PR TITLE
Motion sensor refactor

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Refactored the motion sensor stuff and retain the pedometer in the location manager code.

### CODE REVIEW

File: CRFModuleValidation/CRFMotionRecorder.swift [hunk 1]
```diff
@@ -37,13 +37,41 @@ import ResearchSuite
...
+    /// recorder types then the reference frame is `.xMagneticNorthZVertical`. Otherwise,
+    /// the motion recorder will use `.xArbitraryZVertical`.
     case attitude
+    
+    /// Calculated vector for the direction of gravity in the coordinates of the phone.
+    /// This is included in the `CMDeviceMotion` data object.
     case gravity
-    case gyro
+    
+    /// The magnetic field of the device for devices with a gyro that is included in the
```
Comment (Erin-Mounts) [inline CRFModuleValidation/CRFMotionRecorder.swift]: should it say "devices with a magnetometer"?

**Review by Erin-Mounts — APPROVED** [review]
looks good

**Comment by Erin-Mounts** [comment]
one nit, otherwise looks good--merge when ready

File: CRFModuleValidation/CRFMotionRecorder.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply CRFModuleValidation/CRFMotionRecorder.swift]: oops.

**Pull request merged**